### PR TITLE
fix grammatical and punctuation errors on CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by hossein13m**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

The following Pull request, fix some grammatical issues and add punctuation to improve the readability of the `CONTRIBUTING.md` file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ Before we get started, here are a few things we expect from you (and that you sh
 
 * Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on "how open source is done." Try to listen to others rather than convince them that your way is correct.
 * Open Source Guides are released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.
-* If you open a pull request, please ensure that your contribution passes all tests. If there are test failures, you will need to address them before we can merge your contribution.
-* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created as others will do so if they appreciate it.
+* Please ensure that your contribution passes all tests if you open a pull request. If there are test failures, you will need to address them before merging your contribution.
+* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created, as others will do if they appreciate it.
```
Comment (smashwilson) [inline CONTRIBUTING.md:36]:
```suggestion
* Please ensure that your contribution passes all tests if you open a pull request. If there are test failures, you will need to address them before we can merge your contribution.
* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created, as others will do so if they appreciate it.
```

I'm on board with most of the rephrasing but:

* Contributors don't have permission to merge their own contributions, so changing "we can merge" to "merging" doesn't make sense; and
* I think the second line still reads better with the "so" in place (but adding the comma is 👍🏻 )

**Review by Pakon2543 — COMMENTED** [review]
*  @github/communities-oss-reviewers

File: CONTRIBUTING.md (hunk 1 above)
Comment (hossein13m) [reply CONTRIBUTING.md:36]: All right that's OK.

**Review by hossein13m — COMMENTED** [review]
Checked the comment on pull request

**Review by Ansarbi — COMMENTED** [review]
مشارکت

**Review by smashwilson — APPROVED** [review]
:+1: Thanks!

**Comment by Przemek270189wav** [comment]
gh pr checkout 2772

**Review by JasonF911 — COMMENTED** [review]
fiwnkrh

**Pull request merged**